test/system/250-systemd.bats

```diff
@@ -281,4 +281,30 @@ LISTEN_FDNAMES=listen_fdnames" | sort)
     is "$output" "" "output should be empty"
 }
 
+# https://github.com/containers/podman/issues/13153
+@test "podman rootless-netns slirp4netns process should be in different cgroup" {
+    cname=$(random_string)
+    local netname=testnet-$(random_string 10)
+
+    # create network and container with network
+    run_podman network create $netname
+    run_podman create --name $cname --network $netname $IMAGE top
+
+    # run container in systemd unit
+    service_setup
+
+    # run second container with network
+    cname2=$(random_string)
+    run_podman run -d --name $cname2 --network $netname $IMAGE top
+
+    # stop systemd container
+    service_cleanup
+
+    # now check that the rootless netns slirp4netns process is still alive and working
+    run_podman exec $cname2 ping -c1 8.8.8.8
```

Comment:
Careful... I don't see any other instances of outside-world pings in our tests. ISTR that some test environments (notably RHEL gating) have restricted firewalls that might block ICMP. (But I'm not sure). Is there any way to do this without requiring full Internet connectivity?

Reply:
We already have a nslookup google.com in the tests

Reply:
Changed to nslookup since this already works in the other test.

Reply:
Thanks. `nslookup` is expected to work fine even on networks where ICMP is blocked.